test/liftN.js

```diff
@@ -50,4 +44,25 @@ describe('liftN', function() {
     var addM = R.liftN(2, R.add);
     assert.deepEqual(addM(Maybe(3), Maybe(5)), Maybe(8));
   });
+
+  it('correctly interprets [] as a functor', function() {
+    assert.deepEqual(addN3([1, 2, 3], [1, 2], [1, 2, 3]), [3, 4, 5, 4, 5, 6, 4, 5, 6, 5, 6, 7, 5, 6, 7, 6, 7, 8]);
+    assert.deepEqual(addN3([1], [2], [3]), [6]);
+    assert.deepEqual(addN3([1, 2], [3, 4], [5, 6]), [9, 10, 10, 11, 10, 11, 11, 12]);
+  });
+
+  it('correctly interprets ((->) r) as a functor', function() {
```

Comment:
My understanding is that the return type of the function is the important part.
```javascript
var both = R.liftN(2, R.and);
// both :: Applicative f => f Bool -> f Bool -> f Bool
```
When we're talking about functions as _Applicative Functors_ we interpret 
`f Bool` to mean `((->) r) Bool` or `r -> Bool` 
so one interpretation of the type of the type of `both` is
```haskell
both :: (r -> Bool) -> (r -> Bool) -> (r -> Bool)
```
> Would it be accurate to state that liftN interprets a -> b as a functor? It seems to me we're trying to say that unary functions are functors.

In haskell I'm pretty sure that only unary functions can be applicative functors. In Javascript that's not the case, since we can hack around the arguments object to accept whatever number of arguments we want. We could indicate this by changing the description to something like:
> correctly interprets (* ->) as a functor